iis-wf/iis-wf-export-actionmanager/src/main/java/eu/dnetlib/iis/wf/export/actionmanager/entity/patent/PatentExporterJob.java

```diff
@@ -388,7 +389,7 @@ private static Publication buildOafPublication(Patent patent, String patentIdToE
             result.setCountry(buildOafEntityResultMetadataCountries(patent.getApplicantCountryCodes()));
         }
         
-        result.setInstance(Collections.singletonList(buildOafEntityResultInstance(patent, patentEpoUrlRoot)));
+        result.setInstance(Collections.singletonList(buildOafEntityResultInstance(patent, patentEpoUrlRoot, pids)));
```

Comment:
OK, so I can see we are simply adding pids in the new place. 

I guess we should agree with @claudioatzori whether the duplication is expected or maybe we should avoid setting `payload.pid[]` field which could potentially become deprecated.

Reply:
According to [Claudio's answer](https://issue.openaire.research-infrastructures.eu/issues/6281#note-14) we're staying with duplication for now.